### PR TITLE
configure codecov

### PR DESCRIPTION
**Pull request by danielnorberg**
# Hey, I just made a Pull Request!

## Description
Enable [codecov.io](https://codecov.io/gh/spotify/flo) comments.

## Checklist for PR author(s)

- [ ] Changes are covered by unit test
- [ ] All tests pass
- [ ] Code coverage check passes
- [ ] Error handling is tested
- [ ] Errors are handled at the appropriate layer
- [ ] Errors that cannot be handled where they occur are propagated
- [ ] Relevant documentation updated
- [x] This PR has NO breaking change to public API
- [ ] This PR has breaking change to public API and it is documented

## Checklist for PR reviewer(s)

- [x] This PR has been incorporated in release note for the coming version
- [x] Risky changes introduced by this PR have been all considered

### CODE REVIEW

**Comment by honnix** [comment]
👍

File: .codecov.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+coverage:
+  precision: 2
+  round: down
+  range: 70...100
```
[the remark this answers is not in the document]
Comment (danielnorberg) [reply .codecov.yml]: Heh took this from styx but yeah, what should we set it to?

[the remark this answers is not in the document]
Comment (honnix) [reply .codecov.yml]: No idea. 80 maybe. What is the coverage now?

**Comment by honnix** [comment]
It seems we have merely 70%. Let keep 70 as lower boundary now.

**Pull request merged**